tsconfig.json

```
@@ -3,8 +3,7 @@
   "compilerOptions": {
     "target": "ES2020",
     "lib": ["ES2020"],
-    "module": "ES2020",
-    "moduleResolution": "node",
+    "module": "Node16",
```

```suggestion
    "module": "node16",
```

No need to capitalize it as far as I am aware (and TS doesn’t either?)

---

VSCode or default  shema suggest  `Node16` actually.

---

Why would they? TS doesn’t: https://www.typescriptlang.org/tsconfig#compilerOptions

---

Do you have a source on GH of that you are talking about? Maybe there is an issue, and there is a good (or not) reason they do this.

---

I think there's no technical reason, these are case-insensitive for sure, but, indeed, the [official schema](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/tsconfig.json) wants it in PascalCase, I guess for a smidge of readability and consistency.

---

Meh, I always find it kinda weird to use camelcase/uppercase in places where it isn’t needed? I don’t write `<FigCaption>` or `<!DOCTYPE> in my HTML either 🤷‍♂️.
As the code currently contains `ES` too, I’m fine with this. But I’d prefer in some future moment to do away with unneeded casing!

---

@wooorm See https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/tsconfig.json#L323-L348

---

Against my own preference I suggest to keep it as `Node16`, because it’s more consistent with the above values of `ES2020` instead of `es20202`.

Perhaps we could create a reusable tsconfig for unified to avoid these kind of nits and enforce consistency across all projects.